student/src/main/java/tds/student/sql/repository/remote/impl/RemoteExamRepository.java

```diff
@@ -19,6 +19,7 @@
 import org.springframework.web.client.HttpStatusCodeException;
 import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponents;
```

Comment:
Not sure I follow the changes here.  The underlying code didn't change and I don't see code removal so it doesn't look like it is an import move.  Are these used?